lib/system/gc_regions.nim

```diff
@@ -390,12 +390,14 @@ proc GC_disableMarkAndSweep() = discard
 proc GC_getStatistics(): string = return ""
 
 proc getOccupiedMem(): int =
+  ## Return the currently occupied memory as an integer.
   result = tlRegion.totalSize - tlRegion.remaining
 proc getFreeMem(): int = tlRegion.remaining
 proc getTotalMem(): int =
   result = tlRegion.totalSize
 
 proc getOccupiedMem*(r: MemRegion): int =
+  ## Return the currently occupied memory as an integer.
```

Comment:
It would be nice to add some more detail here, for example:

* Mention that it returns the memory occupied by the *specified memory region*
* Mention what unit it returns, bytes? kilobytes? megabytes? I don't know myself so you'll need to investigate :)

Reply:
It is always in bytes.